### PR TITLE
Fixes AI holopad runetext, right click interact, call reject button

### LINKED ISSUE #39252: Holopad speech isn't logged in the player panel
Round ID: 91315

Server revision compiled on: 2018-07-20

The following pull requests are currently test merged:
#39220: 'Makes having high body temperature more deadly in general' by vuonojenmustaturska at commit 98cfe0e111
#39136: 'Removes can_vv_get from some things' by Cyberboss at commit 75ea5e3c1b
#39118: '[Ready] Communal Resource Storage II' by SpaceManiac at commit c9b4bf3a8d

An AI player used a hologram to talk to someone, and the speech didn't show up in the say logs under the player panel (both client and mob were missing them)

The speech was still logged in the game log for the round.

### LINKED ISSUE #53053: Holopad transmissions should use runechat
People keep ignoring my holopad transmissions because all the local chat they need is available via runechat. I must keep with the times.

### PR DESCRIPTION
**Pull request by Wayland-Smithy**
## About The Pull Request

See changelog.

image
image

## Why It's Good For The Game

Better Holopad functionality.

Fixes: #53053
Closes: #39252 holopad transmissions are in-fact already shown on the player panel:
image

## Changelog
:cl:
qol: Holopad incoming calls can now be rejected.
qol: AI secondary attack now opens Holopad UIs.
fix: Fixed connected Holopad callers not seeing their own runechat text.
fix: Fixed AI hologram transmit runechat text.
fix: Fixed the AI radio help message giving false info about the generic department key.
/:cl:

### CODE REVIEW

**Comment by AlinaStarkova** [comment]
I like this because whenever im AI and holopadding people to talk to them privately they always ignore me because they are conditioned to read runechat not chat

File: code/modules/jobs/job_types/ai.dm [hunk 1]
```diff
@@ -71,3 +71,5 @@
 /datum/job/ai/config_check()
 	return CONFIG_GET(flag/allow_ai)
 
+/datum/job/ai/radio_help_message(mob/M)
+	to_chat(M, "<b>Prefix your message with :b to speak with cyborgs and other AIs.</b>")
```
Comment (Mothblocks) [inline code/modules/jobs/job_types/ai.dm:75]: Undocumented change--put it in the changelog.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/ai.dm (hunk 1 above)
Comment (Wayland-Smithy) [reply code/modules/jobs/job_types/ai.dm:75]: I did tho? `fix: Fixed the AI radio help message giving false info about the generic department key.`

Comment (Mothblocks) [reply code/modules/jobs/job_types/ai.dm:75]: Mistook that for something else, my bad.

Comment (Wayland-Smithy) [reply code/modules/jobs/job_types/ai.dm:75]: No worries. Might have been better for me to describe how the fix is an override.

**Pull request merged**